blocklylib-core/src/main/java/com/google/blockly/android/BlocklyActivityHelper.java

```diff
@@ -100,11 +100,11 @@ public BlocklyActivityHelper(AppCompatActivity activity,
                 .setClipDataHelper(mClipDataHelper)
                 .setWorkspaceHelper(mWorkspaceHelper)
                 .setBlockViewFactory(mBlockViewFactory)
-                .setWorkspaceFragment(mWorkspaceFragment)
```

Comment:
Revert this line.  Per below, there is little benefit to abstracting the WorkspaceFragment.